src/api.js

```diff
@@ -50,14 +50,15 @@ export async function createRank(boardId, name, position, data) {
   });
 }
 
-export async function createBoard(name, data) {
+export async function createBoard(name, data, ice_breaking) {
   return requestJson(`${api_host}/boards`, {
     method: 'POST',
     body: JSON.stringify({
       name,
       data,
       cards_open: true,
       voting_open: true,
+      ice_breaking,
```

Comment:
Does this work as is? I thought we'd need to add support in the backend.
I'm happy to add that if it's needed.

Reply:
Actually, we do need to update the backend as well. There's a merge request there already ;)

Reply:
Done!